app/views/helpers/FullResultsPageHelper.scala

```diff
@@ -366,4 +367,91 @@ object FullResultsPageHelper extends ViewHelper {
     }
 
   }
+
+  private def taxableProfitTable(calculatorResult: CalculatorResult, taxableProfit: Int, distributions: Int)(implicit
+    messages: Messages
+  ): Html = {
+
+    def table(d1: TaxDetails, d2: TaxDetails) = {
+      val totalDays = d1.days + d2.days
+      val taxProfitDistributions = taxableProfit + distributions
+      govukTable(
+        Table(
+          rows = Seq(
+            Seq(
+              TableRow(content = HtmlContent(p(messages("fullResultsPage.taxableProfit.daysAllocated")))),
+              TableRow(content = HtmlContent(d1.days.toString), classes = "govuk-table__cell--numeric"),
+              TableRow(content = HtmlContent(d2.days.toString), classes = "govuk-table__cell--numeric"),
+              TableRow(content = HtmlContent(totalDays.toString), classes = "govuk-table__cell--numeric")
+            ),
+            Seq(
+              TableRow(content = HtmlContent(p(messages("fullResultsPage.taxableProfit")))),
+              TableRow(
+                content = HtmlContent(CurrencyUtils.decimalFormat(BigDecimal(taxableProfit) / totalDays * d1.days)),
+                classes = "govuk-table__cell--numeric"
+              ),
+              TableRow(
+                content = HtmlContent(CurrencyUtils.decimalFormat(BigDecimal(taxableProfit) / totalDays * d2.days)),
+                classes = "govuk-table__cell--numeric"
+              ),
+              TableRow(
+                content = HtmlContent(CurrencyUtils.decimalFormat(taxableProfit)),
+                classes = "govuk-table__cell--numeric"
+              )
+            ),
+            Seq(
+              TableRow(content = HtmlContent(p(messages("fullResultsPage.taxableProfit.distributions")))),
+              TableRow(
+                content = HtmlContent(CurrencyUtils.decimalFormat(BigDecimal(distributions) / totalDays * d1.days)),
```

Comment:
why are we computing the adjusted values on the fly, dont we have methods in calculator result to get it?

If they dont exist, we need to add them

Reply:
The values in the table don't come from back-end those are direct form values (taxable income, distributions, both).  Should we add those to backend?

Reply:
As far as I understand, the values you are displaying are adjust profits and distributions for the tax year, which is automatically computed in the calculator result.

Reply:
Implemented via backend. Intergated with last commit.